limitations.md

```diff
@@ -0,0 +1,13 @@
+# Limitations in iceoryx
+- limited Windows 10 support [\#33](https://github.com/eclipse-iceoryx/iceoryx/issues/33)
+- no support for datatypes with heap or virtual functions for transferring data over iceoryx [\#517](https://github.com/eclipse-iceoryx/iceoryx/issues/517)
```

Comment:
This is closed, but limitation will be added to `overview.md` as it won't be fixed.